domino_cost/cost.py

```diff
@@ -11,25 +13,38 @@
 from dash import dash_table
 from dash.dash_table import DataTable
 from pandas import DataFrame
-from pandas import Timestamp
 
+from domino_cost import constants
 from domino_cost.constants import NO_TAG
 from domino_cost.cost_enums import CostAggregatedLabels
 from domino_cost.cost_enums import CostFieldsLabels
 from domino_cost.cost_enums import CostLabels
 
+logger = logging.getLogger(__name__)
+
 
 def get_domino_namespace(api_host) -> str:
     pattern = re.compile("(https?://)((.*\.)*)(?P<ns>.*?):(\d*)\/?(.*)")
     match = pattern.match(api_host)
+    logger.info("getting domino namespace from %s", api_host)
     return match.group("ns")
 
 
-def get_today_timestamp() -> Timestamp:
-    return pd.Timestamp("today", tz="UTC").normalize()
+def format_date(date_ts):
+    date_obj = date.fromisoformat(date_ts)
+    return date_obj.strftime(constants.DATE_FORMAT)
+
+
+def to_pd_ts(time_ts: str = "today") -> pd.Timestamp:
+    return pd.Timestamp(time_ts, tz="UTC").normalize()
+
+
+def get_last_30() -> date:
```

Comment:
I believe this method is unused. 
If we are to use it, would it make sense to change it to be `get_last_N_days(days: int) -> date`?